package.json

```diff
@@ -16,7 +16,7 @@
     "dev": "tsdx watch",
     "prebuild": "npm run test",
     "build": "tsdx build",
-    "prepublish": "npm run build",
+    "prepublishOnly": "npm run build",
```

Comment:
I completely agree on this since yarn is running all prepublish scripts on install.